contrib/depends/Makefile

```diff
@@ -0,0 +1,227 @@
+.NOTPARALLEL :
+
+SOURCES_PATH ?= $(BASEDIR)/sources
+BASE_CACHE ?= $(BASEDIR)/built
+SDK_PATH ?= $(BASEDIR)/SDKs
+NO_QT ?=
+NO_WALLET ?=
+NO_UPNP ?=
+FALLBACK_DOWNLOAD_PATH ?= https://bitcoincore.org/depends-sources
```

Comment:
Does this mean the build process actually downloads stuff ? If it does, does it actually execute it ?

Reply:
getmonero should make the sources available as well, so we can change the download path from bitcoincore.org to getmonero.org .